nameko/containers.py

```diff
@@ -140,8 +157,13 @@ def __init__(self, service_cls, worker_ctx_cls, config):
         self._died = Event()
 
     @property
+    def interface(self):
```

Comment:
This is an experiment / talking point. The idea is to pass a limited interface of some sort into the extensions, rather than giving them unfettered access to the whole container.